deploy/deploy_delta_upgrade.ts

```diff
@@ -0,0 +1,208 @@
+        deployer,
+        deployments,
+        skipRegister
+    )
+    const controller = await contractDeployer.fetchDeployedController()
+
+    const gitCommitHash = await contractDeployer.getGitHeadCommitHash()
+    const governanceActions = []
+
+    // Deploy contracts
+    await contractDeployer.deployAndRegister({
+        contract: "BondingVotes",
+        name: "BondingVotes",
+        args: [controller.address],
+        proxy: true
+    })
+    governanceActions.push(
+        setContractInfoAction(network, gitCommitHash, "BondingVotesTarget"),
+        setContractInfoAction(network, gitCommitHash, "BondingVotes")
+    )
+
+    const treasury = await contractDeployer.deployAndRegister({
+        contract: "Treasury",
+        name: "Treasury",
+        args: [],
+        proxy: false
+    })
+    governanceActions.push(
+        setContractInfoAction(network, gitCommitHash, "Treasury")
+    )
+
+    const Treasury: Treasury = await ethers.getContractAt(
+        "Treasury",
+        treasury.address
+    )
+
+    // We should already initialize the treasury since it's not a proxy
+    await Treasury.initialize(
+        config.treasury.minDelay,
+        [], // governor will be added as a proposer later
+        [ethers.constants.AddressZero], // let anyone execute proposals
+        deployer // temporary admin role for deployer
+    ).then(tx => tx.wait())
+
+    const livepeerGovernor = await contractDeployer.deployAndRegister({
+        contract: "LivepeerGovernor",
+        name: "LivepeerGovernor",
+        args: [controller.address],
+        proxy: true
+    })
+    governanceActions.push(
+        setContractInfoAction(network, gitCommitHash, "LivepeerGovernorTarget"),
+        setContractInfoAction(network, gitCommitHash, "LivepeerGovernor")
+    )
+
+    const llDeployment = await deployments.get("SortedDoublyLL")
+
+    await contractDeployer.deployAndRegister({
+        contract: "BondingManager",
+        name: "BondingManagerTarget",
+        libraries: {
+            SortedDoublyLL: llDeployment.address
+        },
+        args: [controller.address],
+        proxy: false // we're deploying the Target directly, so proxy is false
+    })
+    governanceActions.push(
+        setContractInfoAction(network, gitCommitHash, "BondingManagerTarget")
+    )
+
+    // Setup/initialize contracts (or print the required governance actions)
+
+    // Grant proposer and executor roles to governor and renounce deployer admin role
+    const roles = {
+        proposer: await Treasury.PROPOSER_ROLE(),
+        canceller: await Treasury.CANCELLER_ROLE(),
+        executor: await Treasury.EXECUTOR_ROLE(),
+        admin: await Treasury.TIMELOCK_ADMIN_ROLE()
+    }
+    for (const role of [roles.proposer, roles.canceller, roles.executor]) {
+        await Treasury.grantRole(role, livepeerGovernor.address).then(tx =>
+            tx.wait()
+        )
+    }
+
+    await Treasury.renounceRole(roles.admin, deployer).then(tx => tx.wait())
```

Comment:
Recommend executing this step only after validating that everything is configured properly.